spec/acceptance/nodesets/docker/ubuntu-14.04.yml

```diff
@@ -0,0 +1,15 @@
+HOSTS:
+  ubuntu-1404-x64:
+    platform: ubuntu-14.04-amd64
+    hypervisor : docker
+    image: ubuntu:14.04
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - 'rm /usr/sbin/policy-rc.d'
+      - 'rm /sbin/initctl; dpkg-divert --rename --remove /sbin/initctl'
```

Comment:
This comes from the upstart Docker hub image https://hub.docker.com/r/dockerimages/ubuntu-upstart/~/dockerfile/

It's required to get upstart to work by being run directly. It removes a fake symlink introduced in the base ubuntu image, then restores the original /sbin/initctl from the package.